storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/webapp/LogviewerResource.java

```diff
@@ -62,6 +64,14 @@
             "logviewer:num-download-log-daemon-file-http-requests");
     private static final Meter meterListLogsHttpRequests = StormMetricsRegistry.registerMeter("logviewer:num-list-logs-http-requests");
 
+    private static final Meter numSearchLogRequests = StormMetricsRegistry.registerMeter("logviewer:num-search-logs-requests");
```

Comment:
I see why it's useful to know how long they take, but is it useful to know how many searches have happened as well?

Reply:
It also reports the frequency of requests handling. I think it can reflect the server load in a way.